tests/testthat/test-pstr_add_reductions.R

```diff
@@ -1,71 +1,82 @@
-test_that("throws an error when the user do not give enough parameters", {
-  expect_error(pstr_add_reductions())
-  expect_error(pstr_add_reductions(pstr_toy_companies()))
-  expect_error(pstr_add_reductions(pstr_toy_companies(), pstr_toy_ep_weo()))
-})
+test_that("with missing obligatory arguments errors gracefully", {
+  companies <- pstr_toy_companies()
+  ep_weo <- pstr_toy_ep_weo()
+  weo <- pstr_toy_weo_2022()
```

Comment:
I'm starting all tests by defining the required datasets. This will minimize the diff when we update the test-data shortly.

Note I'm referring to `weo` not `weo_2022` because the year will inevitably change so the argument should be called just `weo` -- see  #131.